src/interfaces/IPoolManager.sol

```diff
@@ -204,4 +211,6 @@ interface IPoolManager is IProtocolFees, IERC6909Claims {
     /// @param nSlots Number of slots to load into return value
     /// @return value The value of the sload-ed slots concatenated as dynamic bytes
     function extsload(bytes32 slot, uint256 nSlots) external view returns (bytes memory value);
+
+    function getReserves(Currency currency) external view returns (uint256 balance);
```

Comment:
natspec! good to say in it that these are just temporary values not the true reserves at all times

Reply:
+1